tests/smart-feedin.evcc.yaml

```diff
@@ -18,6 +19,18 @@ meters:
     type: template
     template: demo-meter
     power: -2000
+  - name: pv
+    type: custom
+    power:
+      source: js
+      vm: shared
+      script: |
+        8000;
+    feedindisablelimitenable:
```

Comment:
@andig is this the best name? Can we be shorter? `feedinDisable`, `zeroFeedIn`, ..?